app/assets/stylesheets/config/_colors.scss

```diff
@@ -13,7 +13,20 @@
   ////////////////////////////////////////////////////////////////////////////////////
 
   // Base
-  --base-100: #08090a;
+  --base: #08090a;
+
+  --base-a90: #{rgba(#08090a, 0.9)};
+  --base-a80: #{rgba(#08090a, 0.8)};
+  --base-a70: #{rgba(#08090a, 0.7)};
+  --base-a60: #{rgba(#08090a, 0.6)};
+  --base-a50: #{rgba(#08090a, 0.5)};
+  --base-a40: #{rgba(#08090a, 0.4)};
+  --base-a30: #{rgba(#08090a, 0.3)};
+  --base-a20: #{rgba(#08090a, 0.2)};
+  --base-a10: #{rgba(#08090a, 0.1)};
+  --base-a5: #{rgba(#08090a, 0.05)};
```

Comment:
If we wanted for the names of these variables to be more "even", this one could be:

`--base-a05` so that they all have 2 digits. Otherwise, I wonder if there might be some mix-ups between `--base-a50` and `--base-a5` because one ends in a `0` and the other doesn't.